src/components/data/data-utils.js

```diff
@@ -1,13 +1,10 @@
 import _ from 'lodash/fp';
-import { useState } from 'react';
-import { b, div, h, img } from 'react-hyperscript-helpers';
-import { absoluteSpinnerOverlay, ButtonPrimary, Clickable, DeleteConfirmationModal, Select, spinnerOverlay } from 'src/components/common';
-import Modal from 'src/components/Modal';
-import ReferenceData from 'src/data/reference-data';
-import { Ajax } from 'src/libs/ajax';
+import { div, h, img } from 'react-hyperscript-helpers';
+import { Clickable } from 'src/components/common';
+// Removing this import causes test failures related to circular imports.
+import { Ajax } from 'src/libs/ajax'; // eslint-disable-line
```

Comment:
This is super frustrating. Terra UI has a bunch of import cycles. They're handled ok by the build process, which directly consumes ES Modules. But Jest compiles to CommonJS, which is less tolerant of import cycles.

I'm hoping that I can soon get rid of this by breaking up and removing the data-utils module altogether.